sklearn/metrics/scorer.py

```diff
@@ -33,14 +34,15 @@
 
 
 class _BaseScorer(six.with_metaclass(ABCMeta, object)):
-    def __init__(self, score_func, sign, kwargs):
+    def __init__(self, score_func, sign, kwargs, deprecation_msg=None):
```

Comment:
Or even not put `_deprecation_msg` in the `__init__` at all but just as `None` initialized attribute on the `_BaseScorer` class and manually use 

```python
mean_scorer_error._deprecation_msg = "the message"
```

to activate the deprecation.